Makefile

```diff
@@ -72,7 +76,7 @@ test: REBUILD
 	go test -short -tags='debug testing' -timeout=3s $(pkgs) -run=$(run)
 test-v: REBUILD
 	go test -race -v -short -tags='debug testing' -timeout=15s $(pkgs) -run=$(run)
-test-long: clean fmt REBUILD
+test-long: clean fmt vet REBUILD
 	go test -v -race -tags='testing debug' -timeout=300s $(pkgs) -run=$(run)
```

Comment:
I would prefer if vet was placed at the bottom, otherwise there's not going to be a FAIL indicating that you missed something. If it's at the bottom, you can see the output. Maybe we should do the same for fmt so you can see what files got updated. Right now you don't ever see that because it's buried behind the output of 500 tests.